client/state/themes/utils.js

```diff
@@ -46,6 +46,25 @@ export function isPremium( theme ) {
 	return themeStylesheet && startsWith( themeStylesheet, 'premium/' );
 }
 
+/**
+ * Normalizes a theme obtained via the WordPress.com REST API from a Jetpack site
+ *
+ * @param  {Object} theme  Themes object
+ * @return {Object}        Normalized theme object
+ */
+export function normalizeJetpackTheme( theme = {} ) {
+	if ( ! theme.tags ) {
+		return theme;
+	}
+
+	return {
+		...omit( theme, 'tags' ),
+		taxonomies: { theme_feature: map( theme.tags,
```

Comment:
A change here might make this slightly more readable

```
taxonomies: {
    // Map slugs since JP sites give us no names
    theme_feature: map( theme.tags, slug => ( { slug } ) ) 
}
```

Just a nitpick :smile: